sass/partials/_animations.scss

```diff
@@ -0,0 +1,30 @@
+
+.view-animate.ng-enter { 
+	@include transition-property(all);
+  	@include transition-duration(0.4s);
+  	@include transition-timing-function(linear);
+	opacity: 0;
+}
+
+.view-animate.ng-enter-active {
+	@include transition-property(all);
+  	@include transition-duration(0.4s);
+  	@include transition-timing-function(linear);
+  	opacity: 1;
+}
+
+.hide-fade.ng-hide-add, .hide-fade.ng-hide-remove {
+
+  @include transition-property(all);
+  @include transition-duration(0.4s);
+  @include transition-timing-function(linear);
+  display:block!important;
+}
+.hide-fade.ng-hide-add.ng-hide-add-active,
+.hide-fade.ng-hide-remove {
+  opacity:0;
+}
+.hide-fade.ng-hide-add,
+.hide-fade.ng-hide-remove.ng-hide-remove-active {
+  opacity:1;
+}
```

Comment:
try to keep indentation consistent, the first couple blocks look like tabs, while the last couple look like spaces.